work-in-progress/event_finder.ipynb

```diff
@@ -163,8 +170,8 @@
    "outputs": [],
    "source": [
     "# edit path below and filename for shapefile to use\n",
-    "shapefile_filename = 'PajaroRiverWatershed.zip'\n",
-    "spatial_domain = \"Pajaro\" # name will appear in title"
+    "shapefile_filename = \"PajaroRiverWatershed.zip\"\n",
```

Comment:
Not necessarily related to this PR, but I think making this line `shapefile_filename = ...` instead will prompt the user earlier to replace the ellipses with a shapefile that they have, incase they don't have the PajaroRiverWatershed.zip shapefile.

Reply:
Ok replaced the shapefile path and region name with ellipses plus additional instructions.